src/metrics.cc

```diff
@@ -109,6 +109,11 @@ Metrics::Metrics()
                     "execution per-model.")
               .Register(*registry_)),
 
+      model_load_time_family_(prometheus::BuildGauge()
+                                  .Name("nv_model_load_time")
```

Comment:
See my comments on doc - curious if we need to match an exact metric name or not. For example, if the prefix `nv_` is OK or not.

Reply:
Resolved after ack from customer.